src/Button/index.js

```diff
@@ -2,54 +2,140 @@
 import * as React from "react";
 
 import IconWrapper from "./IconWrapper";
-import { colors, fontSizes } from "../constants";
+import Loading from "../icons/Loading";
+import { darken } from 'polished';
```

Comment:
Shouldn't these colours be part of the design token system without requiring an extra library?

Reply:
I understand exactly what you mean, but it's easier to keep it like this. In future it can be helpful, f.e. We don't need to add a new colors for hover and active states whenever is added a new whitelabel.